gio/src/auto/application.rs

```diff
@@ -443,11 +454,6 @@ pub trait ApplicationExt: IsA<Application> + sealed::Sealed + 'static {
         }
     }
 
-    #[doc(alias = "action-group")]
```

Comment:
This actually makes to remove. As the getter was deprecated upstream in 2.3x

Reply:
But the property wasn't marked as deprecated, see https://gitlab.gnome.org/GNOME/glib/-/merge_requests/4101

Reply:
That's there now, you can regenerate this :) Also lots of gtk4-related changes that look relevant